README.md

```diff
@@ -53,7 +53,12 @@ func main() {
         log.Fatalf(err.Error())
     }
 
-    client := opsmngr.NewClient(tc)
+    clientops := opsmngr.SetBaseURL("https://opsmanagerurl/api/public/v1.0/")
```

Comment:
[not-blocking] we have a public const with the public path you can use
```suggestion
    clientops := opsmngr.SetBaseURL("https://opsmanagerurl/" + opsmngr.APIPublicV1Path)
```

Also maybe it would be nice to add a comment (`//`) that if no base url is set the client is set to work with cloud manager but I have no strong opinion here